src/meta/test/JobManagerTest.cpp

```diff
@@ -589,19 +590,17 @@ TEST_F(JobManagerTest, RecoverJob) {
     }
     auto nJobRecovered = jobMgr->recoverJob(spaceId, nullptr);
     ASSERT_EQ(nebula::value(nJobRecovered), 3);
+    std::tuple<JobManager::JbOp, JobID, GraphSpaceID> opJobId;
+    while (jobMgr->jobSize() != 0) {
+      jobMgr->tryDequeue(opJobId);
+    }
   }
 
   // case 2
   // For the balance job, if there are stopped jobs and failed jobs in turn
   // only recover the last balance job
   {
-    std::unique_ptr<JobManager, std::function<void(JobManager*)>> jobMgr = getJobManager();
-    // set status to prevent running the job since AdminClient is a injector
-    jobMgr->status_ = JobManager::JbmgrStatus::STOPPED;
-    jobMgr->bgThread_.join();
-    GraphSpaceID spaceId = 1;
-    int32_t nJob = 3;
-    for (auto jobId = 1; jobId <= nJob; ++jobId) {
+    for (auto jobId = 11; jobId <= 13; ++jobId) {
```

Comment:
Just curious why do you use those fixed id.

Reply:
ok, i'll change it